ledger-core/v2/application/functest/api_test_utils.go

```diff
@@ -125,7 +125,7 @@ func createSimpleWallet() (string, error) {
 	if err != nil {
 		return "", errors.Wrap(err, "failed to unmarshal response")
 	}
-	if resp.Err != nil {
+	if resp.Err != "" {
 		return "", errors.Wrap(err, "problem during execute request")
```

Comment:
Oh, It's my mistake, please fix it. Here and below have to be `errors.Wrap(resp.Err, ... )`

Reply:
With `string` I think `errors.New` or smth else

Reply:
fixed, all 3 cases